README.md

```diff
@@ -104,13 +140,59 @@ The library is also instrumented to use [CMake](https://cmake.org/) which can
 generate a project or Makefiles for a variety of IDE or compiler. For example,
 to generate a Code::Blocks project:
 
-    $ mkdir build
-    $ cd build/
-    $ cmake .. -G"CodeBlocks - Unix Makefiles"
-    
-    c:\> mkdir build
-    c:\> cd build/
-    c:\> cmake .. -G"CodeBlocks - MinGW Makefiles"
+```sh
+$ mkdir build
+$ cd build/
+$ cmake .. -G"CodeBlocks - Unix Makefiles"
+
+c:\> mkdir build
+c:\> cd build/
+c:\> cmake .. -G"CodeBlocks - MinGW Makefiles"
+```
+
+## Runtime envs
```

Comment:
The runtime environment variables are shown in [bin/readme.txt](https://github.com/bacnet-stack/bacnet-stack/blob/master/bin/readme.txt) file. No need to duplicate them here.

Reply:
So, since you have already prepared these instructions, why not put them in the portal doc? One have to go deep into these directories to search for useage?